service_invocation/csharp/http/README.md

```diff
@@ -6,132 +6,170 @@ Visit [this](https://docs.dapr.io/developing-applications/building-blocks/servic
 
 This quickstart includes one checkout service:
 
-- Dotnet client service `checkout` 
+- .NET client service `checkout` 
 
 And one order processor service: 
  
-- Dotnet order-processor service `order-processor`
+- .NET order-processor service `order-processor`
 
-### Run Dotnet order-processor with Dapr
+## Run all apps with multi-app run template file:
 
-1. Open a new terminal window and navigate to `order-processor` directory and install dependencies: 
+This section shows how to run all applications at once using [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) along with `dapr run -f .`.  This is a quick and easy approach to test or debug interactions with multiple applications.  
```

Comment:
```suggestion
This section shows how to run both applications at once using [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) with `dapr run -f .`.  This enables to you test the interactions between multiple applications.  
```

Reply:
Suggested edit. I will debug was easy, but it is not there are many steps you have to do it you actually want to debug Dapr applications. The best option today is https://github.com/man-group/dapr-sidekick-dotnet